getting started/about.md

```diff
@@ -0,0 +1,29 @@
+# About us
+
+## 1. ImagineThis V2: Rapid no-code application builder
+
+Developing a mobile application is a time-consuming and manual process. Projects often require many weeksof development from their prototyping phase before a usable application is produced.
+
+We propose the _ImagineThis_ tool, which takes anannotated high-fidelity prototype of a mobile user interface (UI) designed in Figma and converts it into codethat can run on both _iOS_ and _Android_. We achieved this by developing a Figma file parser, that iterates throughall layers of the Figma file, identifies components and extracts data. Then we use this information to feed intoour code generation algorithms, which output a complete React-Native project that can be compiled and runon both mobile platforms.
+
+Demo Link: https://youtu.be/dLMVbuOB5FA
+
+## 2. Sponsor
+
+Prof. Joseph Anthony Connor, NHS
+
+## 3. Supervisors
+
+Dr. Dean Mohamedally, Dr. Emmanuel Letier, UCL
+
+## 4. Contributors
+
+### Version 2 Team
+
+[Denoy Hossain](https://github.com/dinojs), [Karish Singh](https://github.com/KashSingh1), [Lukas Cerny](https://github.com/lukasotocerny), [Rui Zhou](https://github.com/bbaayezi), [Zhongyuan Li](https://github.com/zhongyuanli), [Xin Deik Goh](https://github.com/DerrickGXD)
+
+### Version 1 Team
```

Comment:
Similarly as with us, just mention that students are part of SSE programme, and summer 2020 industrial project.